scripts/copy_from_upstream/src/kem/family/CMakeLists.txt

```diff
@@ -49,8 +49,8 @@ if(OQS_ENABLE_KEM_{{ family }}_{{ scheme['scheme_c'] }}_{{ impl['name'] }})
         {%- if impl['compile_opts'] %}
     target_compile_options({{ family }}_{{ scheme['scheme'] }}_{{ impl['name'] }} PUBLIC {{ impl['compile_opts'] }})
         {%- endif %}
-        {%- if family == 'classic_mceliece' and impl['name'] == 'avx' %}
```

Comment:
Jinja was missing this line because the implementation name changed from `avx` to `avx2`. I've included the `-O1` option here for proper upstream code integration.